### PR TITLE
Add support for VARBINARY inputs to length Presto function

### PR DESCRIPTION
**Pull request by laithsakka**
Summary: title

Differential Revision: D49731494

### CODE REVIEW

File: velox/functions/prestosql/tests/StringFunctionsTest.cpp [hunk 1]
```diff
@@ -14,8 +14,11 @@
  * limitations under the License.
  */
 #include <gtest/gtest.h>
+#include <velox/vector/BaseVector.h>
```
Comment (mbasmanova) [inline velox/functions/prestosql/tests/StringFunctionsTest.cpp:17]: Do we need these includes?

File: velox/functions/prestosql/tests/StringFunctionsTest.cpp [hunk 2]
```diff
@@ -1797,3 +1800,17 @@ TEST_F(StringFunctionsTest, concatInSwitchExpr) {
       {"This is a long sentence-zzz"_sv, "aaa-This is some other sentence"_sv});
   test::assertEqualVectors(expected, result);
 }
+
+TEST_F(StringFunctionsTest, varbinaryLength) {
+  VectorPtr vector;
+  BaseVector::ensureWritable(
+      SelectivityVector(3), VARBINARY(), pool_.get(), vector);
+  auto flat = vector->asFlatVector<StringView>();
+  flat->set(0, "hi");
```
Comment (mbasmanova) [inline velox/functions/prestosql/tests/StringFunctionsTest.cpp:1809]: Why not use `makeFlatVector<std::string>({"hi", ""}, VARBINARY())`?

**Review by mbasmanova — COMMENTED** [review]
@laithsakka Thank you for adding this capability. Looks great % some comments and

- Please, update PR title to "Add support for VARBINARY inputs to length Presto function" (notice capatalization)
- Please, update documentation

File: velox/functions/prestosql/tests/StringFunctionsTest.cpp (hunk 2 above)
Comment (laithsakka) [reply velox/functions/prestosql/tests/StringFunctionsTest.cpp:1809]: i did not we can do that.

File: velox/functions/prestosql/tests/StringFunctionsTest.cpp (hunk 1 above)
Comment (laithsakka) [reply velox/functions/prestosql/tests/StringFunctionsTest.cpp:17]: those are added by Vscode, we use them directly but they are included from with in other includes, 
happy to remove them. but this was added for example for BaseVector::ensureWriable(...)

File: velox/docs/functions/presto/binary.rst [hunk 3]
```diff
@@ -2,6 +2,10 @@
 Binary Functions
 ================
 
+.. function:: length(binary) -> bigint
```
Comment (mbasmanova) [inline velox/docs/functions/presto/binary.rst:5]: Please, keep functions in alphabetical order.

**Review by mbasmanova — APPROVED** [review]
Thank you, Laith.

**Pull request closed without merging**